package.json

```diff
@@ -4,7 +4,8 @@
   "description": "A directory of local support services and volunteer opportunities",
   "main": "application.js",
   "engines": {
-    "node": ">=8.9.4"
+    "node": "9.5.0",
+    "npm": "6.7.0"
```

Comment:
These were locked down following advice from Heroku about the potential issue with topoJSON, which turned out not to solve the issue, and then it was decided it would be better to have them locked down anyways?

Anything in specific that made you want to go with these versions? Is that what you have locally and it's working, so maybe no reason to try it out with any other versions?

Do we need to update documentation as a result of this change?

Reply:
yes you're exactly right - and yes, these were just the versions locally that were working.

At the moment we don't seem to have documented the versions we should use, at least as far as I could immediately find.  I see various points in the docs where npm/node get installed, so maybe this points to a larger issue about properly documenting our dependencies here - perhaps we could open a ticket on that.

I'm tempted to leave them locked here for the moment and then expand the range we support when needed